jquery.pjax.js

```diff
@@ -310,9 +316,11 @@ function pjax(options) {
     // the last field.
     //
     // http://www.w3.org/html/wg/drafts/html/master/forms.html
-    var autofocusEl = context.find('input[autofocus], textarea[autofocus]').last()[0]
-    if (autofocusEl && document.activeElement !== autofocusEl) {
-      autofocusEl.focus();
+    if(blurFocus) {
+      var autofocusEl = context.find('input[autofocus], textarea[autofocus]').last()[0]
+      if (autofocusEl && document.activeElement !== autofocusEl) {
```

Comment:
Hmm maybe we should also defer the DOM operation until `document.activeElement` is truethy?